go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -202,6 +202,7 @@ func determineOvnkubeRunMode(ctx *cli.Context) (*ovnkubeRunMode, error) {
 	master := ctx.String("init-master")
 	cm := ctx.String("init-cluster-manager")
 	ovnkController := ctx.String("init-ovnkube-controller")
+	ovnkControllerWithNode := ctx.String("init-ovnkube-controller-with-node")
```

Comment:
@npinaeva Can we just use `--init-node --init-ovnkube-controller` to do the same thing? eg why have another flag for this when those two are the same thing?

Reply:
the only difference is that every `--init` flag requires an identity, which is usually a node name, and if you use 2 flags, these identities must be equal or the random one will be chosen. So the better design would be to have an extra flag for identity, and then a list of modes, but that is too many changes. 
Other than that I don't have a preference, 2 flags will be less changes, and 1 new flag guarantees the same identity.